lib/event.rb

```diff
@@ -0,0 +1,99 @@
+# require 'pry'
+#require './lib/standard_diviation'
+
+class Event
+    attr_reader :event_name,
+                :ages
+  def initialize(event_name, ages)
+    @event_name = event_name
+    @ages = ages
+  end
+
+  def name
```

Comment:
Instead of these `name` and `age` methods - I would suggest that you just use the attr_readers that you already have set up.  If you want the names of those methods to be something different, you can initialize the class like this:
```ruby
Class Event
  attr_reader :name, :age
  def initialize(event_name, ages)
    @name = event_name
    @age = ages
  end
end
```